eng/common/pipelines/templates/steps/update-docsms-metadata.yml

```diff
@@ -2,6 +2,9 @@ parameters:
   - name: PackageInfoLocations
     type: object
     default: []
+  - name: PackageInfoPath
```

Comment:
What is different between this and PackageInfoLocations?

Reply:
If I understand correctly one is a list and the other is a single entry. Is there some reason we cannot always use the list?

Reply:
Not all artifacts in ci.yml have `<aritifact-name>.json` file. So the `PackageInfoPath` here is where `*\PackageInfo\` located. We just simple iterate the json files under the folder.